### PR TITLE
refactored the way the inputs display by using 2 forms. A user can cl…

### LINKED ISSUE #23: Fix flask form validation errors on Inputs.
The inputs have errors right after you click begin. errors for inputs are below: 

image

### PR DESCRIPTION
**Pull request by Rolo123y**
- Click a button to show a form and there are 2 buttons to click, one for timeofuse and one for tiered.
- Some visual changes to the inputs on the home page
- Added a 'scripts' folder under app/static which holds any javascript we may need
-->  Currently holds a Jquery file because the minified version doesn't have some features

- Inputs do not include all validations as of yet but still checks for some
--> checks for empty + displays a message
--> checks if a  float value but does not yet display a message

### CODE REVIEW

File: app/routes.py [hunk 1]
```diff
@@ -89,31 +119,44 @@ def renderInputs():
...
 
-    # Go to the inputs section
-    return render_template('home.html',  electricity_form=form, scroll=scroll)
+@app.route('/get_table', methods=["GET", "POST"])
+def get_table():
+    if request.method == "POST":
+        row1 = request.json['row1']
+        row2 = request.json['row2']
+        table_data = [row1, row2]
+        return jsonify(table_data)
```
Comment (Rolo123y) [inline app/routes.py]: can be ignored.

**Comment by gaurav613** [comment]
Can you merge the latest main into this, looks like some files are missing when I try to run it.

**Comment by Rolo123y** [comment]
> 

To run locally, in the __init__.py file, include these items and it should work:
app.config['SQLALCHEMY_DATABASE_URI'] = "somelocallymade.db"
app.config['SECRET_KEY'] = "somekey"

**Review by gaurav613 — APPROVED** [review]
This looks good to me. Once the conflicts are resolved, this can be merged.

**Comment by gaurav613** [comment]
This looks good. Merging now.

**Pull request merged**